lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx

```diff
@@ -4,116 +4,115 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import { faCircle } from '@fortawesome/free-solid-svg-icons';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
-
-import NxThreatIndicator from '../NxThreatIndicator';
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
-import NxToolip from '../../NxTooltip/NxTooltip';
+import NxThreatIndicator, { Props } from '../NxThreatIndicator';
+import { rtlRender, userEvent } from '../../../__testutils__/rtlUtils';
+import { RenderResult, screen } from '@testing-library/react';
+import {
+  allThreatLevelCategories,
+  allThreatLevelNumbers,
+  categoryByPolicyThreatLevel
+} from '../../../util/threatLevels';
 
 describe('NxThreatIndicator', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxThreatIndicator, {});
-
-  it('renders an faCircle NxFontAwesomeIcon with the `nx-threat-indicator` class', function() {
-    const icon = getShallowComponent().find(NxFontAwesomeIcon);
-
-    expect(icon).toHaveClassName('nx-threat-indicator');
-    expect(icon).toExist();
-    expect(icon).toHaveProp('icon', faCircle);
-  });
-
-  it('sets the nx-threat-indicator--unspecified class if no props are passed', function() {
-    const icon = getShallowComponent().find(NxFontAwesomeIcon);
+  const quickRender = rtlRender<Props>(NxThreatIndicator, {});
 
-    expect(icon).toMatchSelector('.nx-threat-indicator--unspecified');
+  it('renders a threat indicator', function() {
+    const el = quickRender(),
+        threatIndicator = el.getByRole('img', { hidden: true, queryFallbacks: true });
```

Comment:
As far as I can tell this isn't hidden

Reply:
> As far as I can tell this isn't hidden

The `svg` rendered on the browser doesn't have `aria-hidden`, but on RTL tests, the `svg` rendered always seem to have `aria-hidden`  even though the `presentational` prop isn't provided.
![image](https://user-images.githubusercontent.com/7110862/214890186-6944b59b-30d1-485b-9c05-12726d376431.png)

Reply:
I was looking into it, and this might be because of Font Awesome's Auto Accessibility feature:
https://cdn.fontawesome.com/help#qa-autoa11y

Reply:
> I was looking into it, and this might be because of Font Awesome's Auto Accessibility feature:
https://cdn.fontawesome.com/help#qa-autoa11y

Well that's not good.  We don't want these icons to be hidden unless `presentational` is set.  And we also don't want behavior in tests to be differing from behavior in the browser, if we can at all help it.  This needs to be investigated more.

Reply:
> > I was looking into it, and this might be because of Font Awesome's Auto Accessibility feature:
> > https://cdn.fontawesome.com/help#qa-autoa11y
> 
> Well that's not good. We don't want these icons to be hidden unless `presentational` is set. And we also don't want behavior in tests to be differing from behavior in the browser, if we can at all help it. This needs to be investigated more.

Added a config to disable auto a11y here: 504d0d9a10e4e39a12b178732a4200baf56b2350

Reply:
> Added a config to disable auto a11y here: https://github.com/sonatype/sonatype-react-shared-components/commit/504d0d9a10e4e39a12b178732a4200baf56b2350

That disables it everywhere.  Is there a way to disable it per FA use?  Maybe just pass in `aria-hidden={false}` explicitly?

Reply:
> That disables it everywhere. Is there a way to disable it per FA use? Maybe just pass in aria-hidden={false} explicitly?

Though I also wonder if it might just be a matter of throwing a `runTimers` in to wait for the tooltips to initialize.  [FA's docs](https://fontawesome.com/v5/docs/web/other-topics/accessibility#svg-with-javascript-semantic-icons) seem to suggest that any icon with a `title` will not be hidden

Reply:
> > That disables it everywhere. Is there a way to disable it per FA use? Maybe just pass in aria-hidden={false} explicitly?
> 
> Though I also wonder if it might just be a matter of throwing a `runTimers` in to wait for the tooltips to initialize. [FA's docs](https://fontawesome.com/v5/docs/web/other-topics/accessibility#svg-with-javascript-semantic-icons) seem to suggest that any icon with a `title` will not be hidden

The runTimers approach does work when dealing with `presentational={false}`. It doesn't work when `presentational={true}` since no tooltips are rendered and there's no `title` attribute. As a result the `aria-hidden` would be auto set to `true`.

I think the best approach is to add `aria-hidden={false}` because in either case when `presentational` is true/false, we don't wait the element to have `aria-hidden={true}`.

I've made the change here: 944940d46065a9f45c8502c910641074bd2b0c02

Reply:
> It doesn't work when presentational={true} since no tooltips are rendered and there's no title attribute. As a result the aria-hidden would be auto set to true.

We _want_ the element to be hidden when `presentational` is true.  Now, it already sort of is due to the `role="presentation"`, but if FA also wants to add `aria-hidden="true"` for that case I have no problem with that.  So I'd revert https://github.com/sonatype/sonatype-react-shared-components/commit/944940d46065a9f45c8502c910641074bd2b0c02

Reply:
> > It doesn't work when presentational={true} since no tooltips are rendered and there's no title attribute. As a result the aria-hidden would be auto set to true.
> 
> We _want_ the element to be hidden when `presentational` is true. Now, it already sort of is hidden due to the `role="presentation"`, but if FA also wants to add `aria-hidden="true"` for that case I have no problem with that. So I'd remove the explicit `aria-hidden={false}`

Removed explicit `aria-hidden` here: bf8ccdd52a2998b01c993a0b7dc6136eae77e18f
Removed auto a11y config here: b85bc20600e5686529105c450302af80c9607d50
Modified tests to use `runTimers` here: 7e322c3dd62ddbe4ec12aa83514365b9ec1309ac